imbi/automations/__init__.py

```diff
@@ -267,8 +267,9 @@ async def retrieve_automations(
     not applicable to the project type, then an exception is raised.
 
     """
+    from imbi import models
```

Comment:
Add a comment about working around circular imports here. Something like `# avoid circular import` is sufficient.